gym_xiangqi/test/xiangqi_env_test.py

```diff
@@ -171,6 +171,11 @@ def test_env_seed(self):
         self.assertIsInstance(seed_list, list)
         self.assertIsInstance(seed_list[0], int)
 
+    def test_env_state_hash_check(self):
+        self.env._state = None
+        with self.assertRaises(AssertionError):
+            self.env.step(300)
+
```

Comment:
Not super necessary, but could you also test by altering the state to an actual 2D array with altered piece locations?

Reply:
Done!

Reply:
I replaced altering piece location with setting the state to `None` since that is a more meaningful test I think.

Reply:
Sounds good!